vgm2txt.c

```diff
@@ -1458,7 +1457,7 @@ static void WriteVGMData2Txt(FILE* hFile)
 					break;
 				if (TempLng >= 0x04)
 					break;
-				sprintf(MinSecStr + TempLng * 0x03, "%02X ", VGMData[VGMPos + TempLng]);
+				snprintf(MinSecStr + TempLng * 0x03, BUFFER_SIZE - TempLng * 0x30, "%02X ", VGMData[VGMPos + TempLng]);
```

Comment:
`- TempLng * 0x30`: is this a typo?
I think it should be `0x03`.

Reply:
since the MinSecStr pointer moves "+ TempLng * 0x03", so the buffer size is reduced to " - TempLng * 0x03"

Reply:
Yes,  but you wrote `0x30` (== 48 decimal), not `3`.